general_tests/session_volume_discount_it_test.go

```diff
@@ -136,7 +136,7 @@ func testSessVolDiscLoadersLoad(t *testing.T) {
 	if err := tSessVolDiscBiRPC.Call(context.Background(), utils.LoaderSv1Load,
 		&loaders.ArgsProcessFolder{
 			// StopOnError: true,
-			Caching: utils.StringPointer(utils.MetaReload),
+			Caching: utils.StringPointer(utils.MetaNone),
```

Comment:
Since the test is important to replicate the production usage, please keep here caching.